### DISCUSSION TITLE
New Domain Address - https://awesome-architecture.com

### DISCUSSION THREAD

**Discussion by mehdihadeli** [mehdihadeli/awesome-software-architecture#37] [Show and tell]
Hi everyone,
awesome software architecture now is available on new domain, [https://awesome-architecture.com/](https://awesome-architecture.com/)

Thanks for your support :)

**Comment by tonivj5** [comment]
![image](https://user-images.githubusercontent.com/7110786/154798255-d6f694dd-dc48-4159-bd80-f7310428d6bf.png)

~~It isn't available yet 😢~~ It works 🚀

**Reply by mehdihadeli** [reply]
Yes, maybe it has some dns problem,
I will fix it

**Reply by tonivj5** [reply]
Cool! By the way, this repo is one of the most **awesome** 😋

**Reply by mehdihadeli** [reply]
@tonivj5 Thanks  ;)

**Reply by mehdihadeli** [reply]
@tonivj5 It's up now.

**Comment by mehdihadeli** [comment]
Site is up and running now 🚀
[https://awesome-architecture.com](https://awesome-architecture.com/)
